src/controllers/employee.js

```
@@ -1,180 +1,606 @@
+    for (let i = 0; i < monthCount; i++) {
+      const d = { id: i + 1, month: months[i], value: 0 };
+      data.push(d);
+    }
+
+    //Iterate through the result
+    for (let result of results2) {
+      const index = result.id - 1;
+      data[index].value = result.value;
+    }
+    //Obtain cumulative values
+    const values = [parseInt(prevYearCount) + parseInt(data[0].value)];
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)